### DISCUSSION TITLE
1.8.0

### DISCUSSION THREAD

**Discussion by alex19EP** [RHVoice/RHVoice#543] [Show and tell]
## What's Changed

* Fix NVDA driver availability without installed voices by @kvark128 in https://github.com/RHVoice/RHVoice/pull/443
* add Polish voice by @grzezlo in https://github.com/RHVoice/RHVoice/pull/445
* Warn about the need to install voices in the NVDA addon by @kvark128 in https://github.com/RHVoice/RHVoice/pull/446
* A build on Linux shouldn't fail without Speech Dispatcher. by @Olga-Yakovleva in https://github.com/RHVoice/RHVoice/pull/454
* mgc with world by @Olga-Yakovleva in https://github.com/RHVoice/RHVoice/pull/459
* voice-building-utils correction and convert-audio to py3 migration. by @grzezlo in https://github.com/RHVoice/RHVoice/pull/460
* Partially parallel context clustering. by @Olga-Yakovleva in https://github.com/RHVoice/RHVoice/pull/465
* Core: handle binary feature flags in the syllabification output. by @Olga-Yakovleva in https://github.com/RHVoice/RHVoice/pull/466
* Voice training: support additional iterations of tree context clustering. by @Olga-Yakovleva in https://github.com/RHVoice/RHVoice/pull/467
* Update Ukrainian dictionary by @VovaMobile in https://github.com/RHVoice/RHVoice/pull/468
* SSML: support the phoneme element, only native phonemes are supported. by @Olga-Yakovleva in https://github.com/RHVoice/RHVoice/pull/469
* Voice-building-utils: save the trim positions in the original recordings. by @Olga-Yakovleva in https://github.com/RHVoice/RHVoice/pull/472
* New male polish voice: Natan by @zstanecic in https://github.com/RHVoice/RHVoice/pull/473
* NVDA addon: Move Rate Boost setting to the conventional position. by @TheQuinbox in <https://github.com/RHVoice/RHVoice/pull/490>
* Segmentation, experimental: use vad to initialize the silence model. by @Olga-Yakovleva in https://github.com/RHVoice/RHVoice/pull/491
* Voice-building-utils: the export-voice command calculates and saves the key parameter. by @Olga-Yakovleva in https://github.com/RHVoice/RHVoice/pull/500
* Core: sentence level tokenization support. by @louderpages in https://github.com/RHVoice/RHVoice/pull/501
* Native data package manager, Android-only for now. by @Olga-Yakovleva in https://github.com/RHVoice/RHVoice/pull/452
* Add script to convert text files to SSML format by @cyrmax in https://github.com/RHVoice/RHVoice/pull/504
* NVDA addon: Make RHVoice compatible with NVDA 2022.1 by @zstanecic in <https://github.com/RHVoice/RHVoice/pull/507>
* Macedonian Language Updates by @louderpages in https://github.com/RHVoice/RHVoice/pull/508
* Fixe segmentation fault (core dumped) on Georgian language by @Olga-Yakovleva in <https://github.com/RHVoice/RHVoice/pull/516>
* Improved support for the reuse of msi package files in SAPI voice installers. by @Olga-Yakovleva in https://github.com/RHVoice/RHVoice/pull/523
* cmake: Support new speech-dispatcher modules location by @living180 in https://github.com/RHVoice/RHVoice/pull/535
* RHVoice-test: don't ignore the quality setting in the configuration file. by @Olga-Yakovleva in https://github.com/RHVoice/RHVoice/pull/538
* NVDA addon: correct the project website url. by @Olga-Yakovleva in https://github.com/RHVoice/RHVoice/pull/539

## New Contributors
* @savoptik made their first contribution in https://github.com/RHVoice/RHVoice/pull/429
* @kvark128 made their first contribution in https://github.com/RHVoice/RHVoice/pull/443
* @TheQuinbox made their first contribution in https://github.com/RHVoice/RHVoice/pull/490
* @louderpages made their first contribution in https://github.com/RHVoice/RHVoice/pull/501
* @cyrmax made their first contribution in https://github.com/RHVoice/RHVoice/pull/504
* @gerazov made their first contribution in https://github.com/RHVoice/RHVoice/pull/534
* @living180 made their first contribution in https://github.com/RHVoice/RHVoice/pull/535

**Full Changelog**: https://github.com/RHVoice/RHVoice/compare/1.6.0...1.8.0

<hr /><em>This discussion was created from the release <a href='https://github.com/RHVoice/RHVoice/releases/tag/1.8.0'>1.8.0</a>.</em>

**Comment by JonnyTech** [comment]
For those looking for the Android build: https://rhvoice.org/download/RHVoice-v1.8.0.apk (16.5Mb)
